framework/tlvf/yaml/tlvf/WSC/eWscVendorId.yaml

```diff
@@ -0,0 +1,9 @@
+#
+---
+_namespace: WSC
+
+eWscVendorId:
+  _type: enum
+  WSC_VENDOR_ID_WFA_1: 0x00
+  WSC_VENDOR_ID_WFA_2: 0x37
+  WSC_VENDOR_ID_WFA_3: 0x2A
```

Comment:
I believe this can be defined as a single enum with storage type uint32_t and constant value 0x00372A that is used in a field with length 3. But endianness should be double-checked.